sql/core/src/test/scala/org/apache/spark/sql/util/DataFrameCallbackSuite.scala

```diff
@@ -205,30 +204,32 @@ class DataFrameCallbackSuite extends QueryTest
       spark.range(10).write.insertInto("tab")
       sparkContext.listenerBus.waitUntilEmpty()
       assert(commands.length == 3)
-      assert(commands(2)._1 == "insertInto")
-      assert(commands(2)._2.isInstanceOf[InsertIntoStatement])
-      assert(commands(2)._2.asInstanceOf[InsertIntoStatement].table
-        .asInstanceOf[UnresolvedRelation].multipartIdentifier == Seq("tab"))
+      assert(commands(2)._1 == "command")
+      assert(commands(2)._2.isInstanceOf[InsertIntoHadoopFsRelationCommand])
+      assert(commands(2)._2.asInstanceOf[InsertIntoHadoopFsRelationCommand]
+        .catalogTable.get.identifier.identifier == "tab")
     }
     // exiting withTable adds commands(3) via onSuccess (drops tab)
 
     withTable("tab") {
       spark.range(10).select($"id", $"id" % 5 as "p").write.partitionBy("p").saveAsTable("tab")
       sparkContext.listenerBus.waitUntilEmpty()
-      assert(commands.length == 5)
-      assert(commands(4)._1 == "saveAsTable")
-      assert(commands(4)._2.isInstanceOf[CreateTable])
-      assert(commands(4)._2.asInstanceOf[CreateTable].tableDesc.partitionColumnNames == Seq("p"))
+      assert(commands.length == 6)
```

Comment:
now we get one more query execution?

Reply:
Because this PR will set the default execution name `command` and https://github.com/apache/spark/blob/7ce7aa47585f579a84dc6dd8f116a48174cba988/sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala#L195 will execute `RepairTableCommand`.